packages/grafana-alerting/src/grafana/notificationPolicies/utils.ts

```diff
@@ -206,6 +206,7 @@ export function convertRoutingTreeToRoute(routingTree: RoutingTree): Route {
           (matcher): LabelMatcher => ({
             ...matcher,
             // sadly we use type narrowing for this on Route but the codegen has it as a string
+            // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
```

Comment:
this was not required before?

Reply:
This was a new additional to our linting rules that was introduced while I was working on this PR – we've moved to eslint instead of betterer and prior to this PR this exception was captured in the betterer results file.